src/config_rc.ts

```diff
@@ -0,0 +1,115 @@
+import * as Controller from './controller'
+import * as Util from './utils'
+import * as Config from './config'
+import * as Native from './native_background'
+import * as Messaging from './messaging'
+import DEFAULTS from './config_defaults'
+import Logger from './logging'
+
+const RC_NAME = 'rc-text'
+const RC_LOC_NAME = 'rc-location'
+const RC_AUTOLOAD_FS_NAME = 'rc-autoload'
+const RC_DEFAULT = `
+# This is the default RC file.
```

Comment:
I could have sworn that someone else mentioned this, but I cannot see it: Comments in VimL are started with ", not #.